lib/gooddata/core/loggers/splunk_logger.rb

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'logger'
+
+module GoodData
+  class SplunkLogger < Logger
+    def hash_to_string(hash)
```

Comment:
info: Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction) (reek)